src/replica/duplication/replica_follower.cpp

```diff
@@ -66,21 +66,27 @@ void replica_follower::init_master_info()
 // ThreadPool: THREAD_POOL_REPLICATION_LONG
 error_code replica_follower::duplicate_checkpoint()
 {
+    zauto_lock l(_lock);
     if (_duplicating_checkpoint) {
         dwarn_replica("duplicate master[{}] checkpoint is running", master_replica_name());
         return ERR_BUSY;
     }
 
     ddebug_replica("start duplicate master[{}] checkpoint", master_replica_name());
-    zauto_lock l(_lock);
     _duplicating_checkpoint = true;
-    async_duplicate_checkpoint_from_master_replica();
+    tasking::enqueue(LPC_DUPLICATE_CHECKPOINT, &_tracker, [=]() mutable {
+        async_duplicate_checkpoint_from_master_replica();
```

Comment:
For review: LPC_DUPLICATE_CHECKPOINT in `DEFAULT_POOL`, when execute _tracker.wait_outstanding_tasks(), it won't be lock